src/codecs.ts

```diff
@@ -871,3 +873,61 @@ export const CookieCsvInput = t.intersection([
 
 /** Type override */
 export type CookieCsvInput = t.TypeOf<typeof CookieCsvInput>;
+
+/// //////////////////////////////////////
+// Transcend Proxy AI Server policies   //
+/// //////////////////////////////////////
+export const Policies = t.literal('redactEmail');
+
+/** Type override */
+export type Policies = t.TypeOf<typeof Policies>;
+
+/** the codec of a route enabled in an AI integration */
+export type EnabledRouteC = t.TypeC<{
+  /** the name of the enabled route */
+  routeName: t.Mixed;
+  /** the enabled policies */
+  enabledPolicies: t.ArrayC<t.LiteralC<Policies>>;
```

Comment:
What's the `C` for in `LiteralC`, `ArrayC`, `AIIntegrationC`, etc?

Reply:
Codec

Reply:
I'm just following the style of `t.ArrayC`, `t.StringC`, etc.

Reply:
Fair enough! I don’t know io-ts very well.